index.html

```diff
@@ -19,7 +19,13 @@
         <nav class="fas fa-bars"></nav>
     </header>
     <section class="home-page-section">
-        <h1 class="Right Intro">Namaste!<br> I'm Ernest</h1>
+        <header class="toolbar second">
+            <a href="#" class="logo">My Logo</a>
+            <nav class="header1 portfolio1">Portfolio</nav>
+            <nav class="header1 about1">About</nav>
+            <nav class="header1 contact1">Contact</nav>
+        </header>
+        <h1 class="Right Intro">I'm Ernest <br>Glad to see you!</h1>
```

Comment:
- [ ] Kindly fix the UI as the minimum view is not in accordance with the Figma view.
- [ ] Make these changes in the card section too and others.
## Your Design
![image](https://user-images.githubusercontent.com/86200356/193257390-5357235e-6210-4f43-9350-40f536910112.png)
## Figma
![image](https://user-images.githubusercontent.com/86200356/193257644-8a7debca-1b66-4d8e-971e-6ba6ed6d43aa.png)